test/state-state.js

```diff
@@ -229,22 +237,22 @@ test('test state-state', function(tt) {
 /*
 stateState.guaranteeAllStatesExist
 */
-test('guaranteeAllStatesExist', function (t) {
-	var ss = null
+test('guaranteeAllStatesExist', t => {
+	let ss = null
 	t.plan(2)
 
 	ss = StateState()
 	ss.add('a', 'hahaha')
 	ss.add('a.b', 'rofl')
 	ss.add('a.b.c', 'um')
-	t.doesNotThrow(function () {
+	t.doesNotThrow(function() {

```

Comment:
Is there a reason this function and the one 7 lines later are not using arrow functions?

Reply:
Nah, I just didn't spend much time on the test files, I mostly focused on the main code.